lib/rubocop/cop/rspec/leaky_constant_declaration.rb

```diff
@@ -121,6 +124,16 @@ def on_module(node)
         def inside_describe_block?(node)
           node.each_ancestor(:block).any? { |ancestor| spec_group?(ancestor) }
         end
+
+        def defined_on_explicit_namespace?(node)
+          if node.is_a?(RuboCop::AST::ClassNode) ||
+              node.is_a?(RuboCop::AST::ModuleNode)
+
+            node.loc.name.source.include?('::')
+          elsif node.is_a?(RuboCop::AST::CasgnNode)
```

Comment:
If the namespace is a cbase, doesn’t it guarantee that there’s `::` in the const name?

Reply:
const_name won't include the leading `::` in this case, that's why we check for cbase explicitly